origami.json

```
@@ -1,5 +1,5 @@
 {
-	"description": "Advertising",
+	"description": "Enables advertising from Googles DFP Ad server, and provides customised demographic, behavioural (via Krux), and contextual (via Admantx) targeting.",
```

Missing apostrophe but otherwise good 👍 
```suggestion
	"description": "Enables advertising from Google's DFP Ad server, and provides customised demographic, behavioural (via Krux), and contextual (via Admantx) targeting.",
```

---

Ugh yep just realised this was here already, but maybe worth fixing here and in README